### ISSUE TITLE
Figure out days / times that work for the most people

### ISSUE THREAD

**Issue by ben-denzer** [FCC-Las-Vegas/meetings#1]
We have tried Saturday mornings and Saturday evenings, they both worked out pretty well. I know at least one person said Tuesday and Wednesdays are best - That would work for me too.

**Comment by 149203** [comment]
I'm not a lock, but weekend mornings or afternoons would be best for me. Would also be interested in ride-sharing. I'm in the northeast.

**Comment by jacobtipp** [comment]
I know vegasJS holds their meetings on Tuesdays whenever they are scheduled. Even though they aren't every month it's probably best not to clash with those dates. Other than that I'm available most weekends.

**Comment by ben-denzer** [comment]
VegasJS is the 3rd Tuesday of each month -  React is every two weeks on Wednesdays, You're right we wouldn't want to clash with that but we could pretty easily avoid those times. - I'm sure there are others too

**Comment by abrahamlaria** [comment]
I really want to attend the meetings but Saturday morning is really difficult for me since I have to take care of my little one.  Any day from Monday to Friday after work would be perfect in my case. But I'll try to adjust to whatever you guys decide.

**Comment by abrahamlaria** [comment]
Maybe we can put a poll with different options

**Comment by ben-denzer** [comment]
Yeah, I think a poll would be a good idea, but the final decision will have to go to Jai - you have any polling app made or ideas how to get that going?

**Comment by jacobtipp** [comment]
Would this poll work? 

http://www.poll-maker.com/poll733278x0B7546a5-30

**Comment by abrahamlaria** [comment]
I don't have any made by me but I think there is an official Facebook poll app.

**Comment by abrahamlaria** [comment]
Not very sure about the "official" but there are plenty of poll apps on FB.

**Comment by jacobtipp** [comment]
@abrahamlaria, good call, I just made a poll in the facebook group

**Comment by ben-denzer** [comment]
@traducer - That would be fine, but it looks like tue - sat are our only options.

**Comment by j7an** [comment]
Innevation Center just notified me that they're changing their schedule to only Tue. - Thur. evenings starting in August. We can certainly try not to clash with the other groups, but the Weekends might have to be another venue starting in August.

**Comment by 149203** [comment]
Iron Yard has been pretty receptive to coding meetups.

**Comment by j7an** [comment]
Iron Yard and Free Code Camp are competitors. Would they be receptive?

**Comment by jacobtipp** [comment]
very good point @j7an

**Comment by 149203** [comment]
Iron Yard has recommended FCC to its graduates. Some of them are on FCC now. We could be seen as facilitating their graduates' growth. I can ask Gabe the next time I'm up there or shoot him an email. I'll do that and check back in.

**Comment by 149203** [comment]
Or we could simply call ourselves a beginner's coding group... :) There was a group last year at this time that was pretty fun called Hipster Hackers. https://github.com/hipsterhackers/courses If we had a name like that... Regardless I'll approach the Iron Yard and come back with something tangible.

**Comment by j7an** [comment]
Thanks. Go ahead and ask Gabe directly. There's no need to beat around the bush. Yea, Hipsterhackers was pretty good. They didn't continue with more classes which is a bummer. I went to one and the instructor was trying to help debug one of the student's code, except she wrote it all in Notepad, no syntax highlighting or code hinting, LOL.

**Comment by 149203** [comment]
Email sent. Hey I wrote my first scripts in notepad! :smile:

**Comment by jacobtipp** [comment]
Yeah I started out with notepad too lol.

Then moved on to sublime.

Now I'm one of those vim/tmux hipsters.

**Comment by 149203** [comment]
Monday morning I'll shoot another email to Iron Yard, and also emails to [Work In Progress](http://www.workinprogress.lv/) and [Co-Operate](https://liquidspace.com/US/NV/las-vegas/co-operate-on-main)

We discussed how cool it would be to have our meetups in a place where startups are working out of. People work with who they know and if we could be who they know...

**Comment by j7an** [comment]
Great meetup we had this past Saturday. I'll create a poll for the new Innevation schedule on Facebook. Duece is suggesting we have a social meetup so everyone can get to know each other. I'll create a poll for that too.

BTW, I just went on Co-Operate's web site and it forwarded me to another site telling me my Mac is infected with malware and that I should call their 800 number, LOL. Gotta love those phishing scammers. Mike, you might want to let Co-Operate know their site maybe hacked.

**Comment by 149203** [comment]
Posted results of today's adventure here: https://www.facebook.com/groups/free.code.camp.las.vegas.nv/permalink/1131678593520283/
